services/directoryServices/__tests__/ResourceDirectoryService.spec.js

```diff
@@ -284,21 +287,26 @@ describe("Resource Directory Service", () => {
         fileName: INDEX_FILE_NAME,
         directoryName: `${resourceRoomName}/${newDirectoryName}`,
       })
+      expect(getResourceSpy).toHaveBeenCalledBefore(mockGitHubService.update)
```

Comment:
Not sure how useful it is that the util function is called before the update - I think we might want to check that the `rename` operation from `baseDirectoryService` is called before `update` operation from `GitHubService`

Reply:
thanks for the catch! updated accordingly